modules/EvseV2G/charger/ISO15118_chargerImpl.cpp

```diff
@@ -184,7 +184,12 @@ void ISO15118_chargerImpl::handle_session_setup(std::vector<types::iso15118::Pay
         }
     }
 
-    if (supported_certificate_service) {
+    const auto pnc_enabled = ((v2g_ctx->evse_v2g_data.payment_option_list[0] == iso2_paymentOptionType_Contract) or
+                              (v2g_ctx->evse_v2g_data.payment_option_list[1] == iso2_paymentOptionType_Contract))
+                                 ? true
+                                 : false;
```

Comment:
`a_bool ? true : false` is equal to `a_bool`
```suggestion
    const auto pnc_enabled = (v2g_ctx->evse_v2g_data.payment_option_list[0] == iso2_paymentOptionType_Contract) or
                             (v2g_ctx->evse_v2g_data.payment_option_list[1] == iso2_paymentOptionType_Contract);
```